src/blocks/seify/config.rs

```diff
@@ -31,6 +32,27 @@ impl Config {
         Pmt::Any(Box::new(self.clone()))
     }
 
+    /// Generate a [`Pmt`] that can be serialized
+    pub fn to_serializable_pmt(&self) -> Pmt {
```

Comment:
Not a huge fan of this name, but needed to be able to distinguish between the `to_pmt` method whose result is *not* serializable over a remote connection.

Reply:
Another option is to replace the `to_pmt` method with this one. The `Config::try_from(Pmt)` function supports both forms.

Reply:
Was the `Box`-ed variant there for performance, convenience, or some other requirement?

> Another option is to replace the `to_pmt` method with this one. The `Config::try_from(Pmt)` function supports both forms.

Was the `Box`-ed variant there for performance, convenience, or some other requirement?

Reply:
IIRC, `Any` requires a `Box`. I wouldn't know of another way. This variant can be used within the same process, i.e., casting it back to `Config`, instead of serialization and deserialization.

Reply:
That part makes sense. It was more of a question of whether this is the right approach to supporting both modalities (in-process vs. inter-process). Probably fine for now.